.travis.yml

```diff
@@ -3,9 +3,10 @@ sudo: false
 language: php
 php:
   - 7.1
+  - 7.2
 
 before_script:
-  - composer install --no-interaction -o
+  - composer install --no-interaction
```

Comment:
Why do you say this flag is useless?

Reply:
I'm sorry and I misunderstand optimized autoloader option.
I add this `-o` back.